app/views/pages/processing_completed_form_submissions.html.md

```diff
@@ -0,0 +1,53 @@
+
+For example, MS Outlook allows you to [set up forwarding rules based on whether the email includes a specific keyword](https://support.microsoft.com/en-gb/office/use-rules-to-automatically-forward-messages-45aa9664-4911-4f96-9663-ece42816d746).
+
+Bear in mind that MS Outlook forwarding rules will pick up any use of a keyword in the submission email - so MS Outlook could forward an email to the wrong place if a person uses the keyword in a way you do not expect. This is less of an issue with questions where you’re asking users to choose options from a list, because you can predict what text will appear in the submission email.
+
+## Get form submissions as JSON or CSV email attachments
+
+If you’re planning to automate your form processing using a low code platform like MS Power Automate or Google Apps Script, it may be easier to work with submission data in JSON or CSV format.
+
+When you create or edit a form, you can opt to get submission data attached to the emails as a JSON or CSV file - as well as in the body of the email. Send yourself test submissions so you can see how emails are laid out and help set up your automation. You can view the [JSON schema for form submissions](https://www.forms.service.gov.uk/json-submissions/v1/schema).
+
+### Automate processing of email submissions
+
+If you’re not sure whether your organisation supports use of a low code platform - or you need help setting up an automation - contact your organisation’s digital or IT team. Some organisations have a team which specialises in low code automation.
+
+If your organisation supports MS Power Automate, the Microsoft website has [guidance on getting started with MS Power Automate](https://learn.microsoft.com/en-us/power-automate/flow-types). Or the Google website has [guidance on getting started with automations in Google Apps Script](https://developers.google.com/apps-script/quickstart/automation).
+
+You can also [join the cross-government low code community](https://www.gov.uk/service-manual/communities/low-code-community).
+
+## Receive form submissions in an AWS S3 bucket
+
+If your organisation has access to the AWS stack, you can opt to receive form submissions in an AWS S3 bucket instead of by email. Submissions are written to the S3 bucket as individual JSON or CSV files.
+
+You’ll need a software engineer to configure and maintain the S3 bucket for you - and to write and maintain code to extract the submission data from the S3 bucket and import it into your CRM system (or whatever system you’re using to do the processing).
+
+If you want to receive form submission in an AWS S3 bucket, [send us a support request](https://www.forms.service.gov.uk/support) confirming that you:
+
+* have a technical team with access to the AWS stack, who can set up and maintain an S3 bucket on your behalf
```

Comment:
```suggestion
```

If the point we're trying to get across that, GOV.UK Forms will not create and maintain an S3 bucket on their behalf.

Frame it as what they need to do:

- You need create and configure an S3 bucket following relevant NCSC guidance...